docs/podman-load.1.md

```diff
@@ -1,22 +1,24 @@
 % podman-load(1)
 
 ## NAME
-podman\-load - Load an image from docker archive
+podman\-load - Load an image from a container image archive into container storage
 
 ## SYNOPSIS
-**podman load** [ARCHIVE]
+**podman load** [*name*[:*tag*]]
 
 ## DESCRIPTION
-**podman load** copies an image from either **docker-archive** or **oci-archive** stored
-on the local machine. **podman load** reads from stdin by default or a file if the **input** flag is set.
-The **quiet** flag suppresses the output when set.
+**podman load** loads an image from either an **oci-archive** or **docker-archive** stored on the local machine into container storage. **podman load** reads from stdin by default or a file if the **input** option is set.
+You can also specify a name for the image if the archive does not contain a named reference.
```

Comment:
I still find the name option confusing. It is always ignored AFAICT (probably because my saved archive includes a name). Would this make more sense?
```
If the archive does not include a name (FIXME: why would that be? Is that OCI vs Docker format?), you can specify a **name** and optional **tag**; these will be silently ignored if the archive already specifies them.
```
(And, in the spirit of not silently ignoring, would it be possible for the code to check for this condition, and say "I am completely ignoring the name:tag you gave me"?)

Reply:
I don't know why this happens, and could not figure it out from the code.  Who coded this up originally?